packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.stories.tsx

```diff
@@ -50,11 +61,12 @@ const Story: FC<{ content: string }> = ({ content }) => {
     return null;
   }
 
+  // TODO Wire up the toolbar state to the editor properly
```

Comment:
Since the toolbar supposedly will become part of the markdown extension, is there a reason to have a separate Toolbar story that needs to wire everything up separately?

Reply:
The toolbar should be optional -- i.e., the extension should not depend on it, so it felt right to keep the story separate as with comments.